changelogs/unreleased/gh-9917-speed-up-transaction-queue-processing.md

```diff
@@ -0,0 +1,3 @@
+## bugfix/raft
```

Comment:
Sorry, I just realized - in the commit message lets change the topic from `raft:` to `limbo:`. Raft didn't change here. It is located in `src/lib/raft` and in `src/box/raft.h+c`.

Reply:
Fixed.